src/styling/mapStyles.js

```diff
@@ -0,0 +1,111 @@
+
+export const mapStyles = [
+    { elementType: "geometry", stylers: [{ color: "#ebe3cd" }] },
+    { elementType: "labels.text.fill", stylers: [{ color: "#523735" }] },
+    { elementType: "labels.text.stroke", stylers: [{ color: "#f5f1e6" }] },
+    {
+      featureType: "administrative",
+      elementType: "geometry.stroke",
+      stylers: [{ color: "#c9b2a6" }],
```

Comment:
do we want colors in a constant file for default colors? at least for the ones we will reuse